utils/stubs.go

```diff
@@ -2,6 +2,7 @@
 
 package utils
 
+// ExperimentalBuild is only for experimental check by Jenkins by now.
```

Comment:
This isn't really accurate as it is used in `docker info`.

perahps: `ExperimentalBuild is a stub which always returns false for builds that do not include the "experimental" build tag`